source/MRMesh/MRStringConvert.cpp

```diff
@@ -50,6 +51,12 @@ std::u8string systemToUtf8( const char* system )
     return asU8String( system );
 #endif
 }
+#else
+std::string systemToUtf8( const char* system )
+{
+    return system;
```

Comment:
The return type may be std::string here, but the conversion inside must remain (for Windows)